optimade/filtertransformers/mongo.py

```diff
@@ -208,3 +199,29 @@ def __default__(self, data, children, meta):
         raise NotImplementedError(
             f"Calling __default__, i.e., unknown grammar concept. data: {data}, children: {children}, meta: {meta}"
         )
+
+    def _recursive_expression_phrase(self, arg):
```

Comment:
Is there a particular reason for not simply making _this_ the `expression_phrase` method? Did you have multiple uses of this helper method in mind?

Reply:
The reason is that it wasn't working X) Lark does something with these methods that mean they can't be called directly or recursively. If you know a way around this then it would certainly make the code cleaner!